src/data-access/repositories/interfaces/MediaRepository.ts

```diff
@@ -1,5 +1,6 @@
 import { Media } from '../../../model/entities/media'
 
 export interface MediaRepository {
-  getMedia(mediaId: string): Promise<Media | undefined>
+  getMedia(): Promise<Media[]>
+  getMediaById(mediaId: string): Promise<Media | undefined>
```

Comment:
It's a bit of a weird scenario, there is actually a unique ID connected to `mediaObjects` but we also use the term `mediaId` because that is how we identify media by their filename (without the file extension). Technically it would be `getMediaByMediaId`, but i don't mind seeing if we can just make do by abstracting it away like this.

Reply:
What is the difference between `Media.id` and `Media.mediaId`?

Reply:
It can be seen here, the `media.id` is the collection it stems from suffixed by a unique identifier.
The `media.mediaId` is the filename without the file extension. 

![image](https://github.com/tv2/sofie-server/assets/32238291/19282b6f-d349-48bd-bf54-b6f1786f9e20)

Reply:
Don't mind the `to2` in the second row, that was just me testing some update-change scenarios :-D

Reply:
It just seems weird that we have an Entity called `Media` that has an attribute that is called `MediaId` but is in fact not the id of the Media...

Reply:
Is `MediaId` the correct name to use? We do have the option to change it.

Reply:
After we discussed in a call we enden up deciding to rename `mediaId` to `mediaName`.

Reply:
Just question:
Even after changing the name of the identify to `mediaName` doesn't the same problem remain, now we have `Media.mediaName` . Is there have any problem to be only `Media.name`?

Reply:
Also a good point, but in the meantime we renamed it to `sourceName` :-D

Reply:
I have updated it be using `sourceName` instead `mediaId`, but the change for introducing `sourceName` is in this PR https://github.com/tv2/sofie-server/pull/84

The endpoint still works without the changes of that PR.